core/src/logger/u_logger.cpp

```diff
@@ -27,6 +27,9 @@ ULogger::ULogger(QObject* parent) :
         case DataTypes::LogType::FULL :
```

Comment:
```suggestion
        case DataTypes::LogType::FULL :
        case DataTypes::LogType::FULLAREA :
```
you can combine these two cases since they do the same thing